src/main/java/org/trigon/hardware/misc/leds/CANdleLEDStrip.java

```diff
@@ -0,0 +1,212 @@
+package org.trigon.hardware.misc.leds;
+
+import com.ctre.phoenix.led.*;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.util.Color;
+import org.trigon.hardware.RobotHardwareStats;
+
+import java.util.function.Supplier;
+
+public class CANdleLEDStrip extends LEDStrip {
+    private static CANdle CANDLE;
+    private static int LAST_CREATED_LED_STRIP_ANIMATION_SLOT = 0;
+    private final int animationSlot;
+    private AddressableLEDStrip simulationLEDStrip;
+    private boolean alternateColor = true;
+    private double lastAlternateColorTime = 0;
+
+    public static void setCandle(CANdle candle) {
+        CANDLE = candle;
+    }
+
+    public CANdleLEDStrip(boolean inverted, int numberOfLEDs, int indexOffset) {
+        super(inverted, numberOfLEDs, indexOffset);
+        LAST_CREATED_LED_STRIP_ANIMATION_SLOT++;
+        animationSlot = LAST_CREATED_LED_STRIP_ANIMATION_SLOT;
```

Comment:
Shouldn't this apply before incrementing?

Reply:
This is how Triumph did it, I'll check it I can swap the order in the real robot